src/main/java/core/basesyntax/service/FileReader.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class FileReader {
```

Comment:
```suggestion
public class FileReaderService {
```
and also add a corresponding interface